_posts/2017-04-30-React-Native-for-iOS-devs.markdown

```diff
@@ -0,0 +1,671 @@
+---
+layout: epic
+title: Intro to React Native for an iOS Developer
+date: 2017-05-30
+categories: [Technology, emission, react-native, react, javascript]
+author: orta
+series: React Native at Artsy
+css: what-is-react-native
+---
+
+React Native is a new native library that vastly changes the way in which you can create applications. The majority of the information and tutorials on the subject come from the angle of _"you are a web developer, and want to do native"_.
```

Comment:
Agreed. Besides the age of 2 years, I’m sure many also still perceive it as new as long as it’s < v1.